src/Core/Core.php

```diff
@@ -70,20 +71,26 @@
 $requestURL = isset($_REQUEST['url']) ? trim($_REQUEST['url'], '/') : false;
 $flush = (isset($_GET['flush']) || $requestURL === trim(BASE_URL . '/dev/build', '/'));
 
-global $manifest;
-$manifest = new ClassManifest(BASE_PATH, false, $flush);
+// Manifest cache factory
+$manifestCacheFactory = new ManifestCacheFactory([
+    'namespace' => 'manifestcache',
+    'directory' => getTempFolder(),
```

Comment:
What if you need *additional* args like a Redis/Memcached url and port? I guess that can wait until we have an app object, and a way to create custom `index.php` (where you could define those services prior to their use, and add them to the injection container manually - without relying on YAML configuration.